src/vstart.sh

```diff
@@ -1642,9 +1642,8 @@ if [ $with_jaeger -eq 1 ]; then
   -p 16686:16686 \
   -p 14268:14268 \
   -p 14250:14250 \
-  jaegertracing/all-in-one:1.20
-fi
-
+  quay.io/jaegertracing/all-in-one
```

Comment:
i cannot find the reason why we need this change in the commit message.

Reply:
in servers like sepia labs, with users might hit  [rate limiting for docker pull](https://blog.container-solutions.com/dealing-with-docker-hub-rate-limiting) it is better to use quay image to avoid this issue

Reply:
could you please note this in the commit message?